browser/src/UI/components/QuickInfo.tsx

```diff
@@ -3,6 +3,79 @@ import * as os from "os"
 import * as React from "react"
 import styled, { boxShadowInset, css, fontSizeSmall, withProps } from "./common"
 
+const markedCss = css`
```

Comment:
The changes for the rendering look great to me! Thanks for all your work here. I wonder if we could split this up into two changes? It seems like the synchronizing highlight groups has some challenges, but we could at least bring in the rendering with the `editor.tokenColors` we have today